src/Controller/GalleryAdminController.php

```diff
@@ -56,4 +48,12 @@ public function listAction(Request $request): Response
             'csrf_token' => $this->getCsrfToken('sonata.batch'),
         ]);
     }
+
+    protected function addRenderExtraParams(array $parameters = []): array
```

Comment:
I plan to make the remove/make final `addRenderExtraParams`/`renderWithExtraParams` as explained https://github.com/sonata-project/SonataAdminBundle/pull/7243#issuecomment-857422324.

What would be the solution here to avoid overriding the method then ?

Reply:
add those parameters to the actions directly, we create 2 duplicate parameters, but even with that, probably less lines of code. should I go for it?

Reply:
I would say we should not pass `persistent_parameters`, and use directly `admin.persistentParameters` in the twig.

And I was wondering if we couldn't use a twig extension instead for `media_pool.contexts`.
Btw there is a class `final class GlobalVariables`, which exposed 
```
    /**
     * @return Pool
     */
    public function getPool()
    {
        return $this->container->get('sonata.media.pool');
    }
```
does it means the pool variable is already passed, as a global variable, to all the twigs ?

Reply:
Done, all was only 2 variables in a single template 😅